rezolvari probleme.js

```diff
@@ -0,0 +1,71 @@
+//problema 1
+function SumVect(vect)
+{
+    var sum = 0;
+
+    for(i=0; i < vect.length; i++)
+    {
+        sum += vect[i];
+    }
+
+    return sum;
+}
+
+//problema 2
+function Get2LargestNo(vect)
+{
+    var numbers = [];
+    numbers[0] = -Infinity;
+    numbers[1] = -Infinity;
+
+    for(i=0; i < vect.length; i++)
+    {
+        if(numbers[1] < vect[i])
+        {
+            numbers[0] = numbers[1];
+            numbers[1] = vect[i];
+        }
+    }
+
+    return numbers;
+}
+
+//problema3
+function ThereAreDuplicate(vect)
+{
+    var suma1 = SumVect(vect);
+	//console.log('suma1 : ' + suma1);
+	//console.log('vect length : ' + vect.length);
+    var suma2 = vect.length * ((vect.length + 1) / 2);
+	//console.log('suma2 : ' + suma2);
+    return !(suma2 == suma1);
+}
+
+function assert( expression, description ) { 
+	var assertStatus = expression ? 'pass' : 'fail'; 
+    console.log(description + ' : ' + assertStatus);
+}; 
+
+
+// unit testing
```

Comment:
This is not proper unit testing. A nice framework for unit testing in javascript is Jasmine.js. A quick google search for jasmine tutorials gave me this result: http://evanhahn.com/how-do-i-jasmine/ . Take a look at that and see if you like how it works. If you don't like it, you should try to find another framework, but as it is these are not proper tests.